test/Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests/MachineKeyTests.cs

```diff
@@ -86,7 +87,7 @@ public void UnprotectWithPurposes(params string[] purposes)
         Assert.Equal(plaintext, result);
     }
 
-    private static IDataProtectionProvider BuildProvider(params string[] purposes)
+    private static IDataProtectionProvider BuildProvider(params IEnumerable<string> purposes)
```

Comment:
this is due to some overload resolution changes with c# version change. pretty cool to use a params with IEnumerable<> though